The fix for
15476 csh should be mediated in the packaging
didn't update the sparc Makefile, so we do it here

diff --git a/usr/src/cmd/csh/sparc/Makefile b/usr/src/cmd/csh/sparc/Makefile
index f28a908376..af8c8c25e8 100644
--- a/usr/src/cmd/csh/sparc/Makefile
+++ b/usr/src/cmd/csh/sparc/Makefile
@@ -74,7 +74,12 @@ $(CSH_PROG): $(COMMONOBJS) $(LOCALOBJS) $(MAPFILES)
 
 $(HDDEP): ../sh.tconst.h
 
-install: all $(ROOTBINPROG)  $(ROOTPROG)
+$(ROOTPROG) := INSLINKTARGET=../has/bin/$(PROG)
+
+$(ROOTPROG):
+	$(INS.symlink)
+
+install: all $(ROOTHASBINPROG) $(ROOTPROG)
 
 clean:
 	$(RM) $(LOCALOBJS) $(COMMONOBJS)
